tests/unit/Bridge/Symfony/Console/RunCommandJobLauncherTest.php

```diff
@@ -20,7 +24,37 @@ public function testLaunch(): void
         $commandRunner->runAsync('yokai:batch:run', 'test.log', $arguments)
             ->shouldBeCalledTimes(1);
 
-        $launcher = new RunCommandJobLauncher(new JobExecutionFactory(), $commandRunner->reveal(), 'test.log');
+        /** @var JobExecutionStorageInterface|ObjectProphecy $storage */
+        $storage = $this->prophesize(JobExecutionStorageInterface::class);
+
+        $storage->store(
+            Argument::that(
+                function ($jobExecution) use ($config) {
```

Comment:
There is no need for $config to be passed to that closure. As this is a test, you can just use the raw value in the assertion